x/authz/keeper/keeper.go

```diff
@@ -118,7 +118,8 @@ func (k Keeper) DispatchActions(ctx context.Context, grantee sdk.AccAddress, msg
 
 			grant, found := k.getGrant(ctx, skey)
 			if !found {
-				return nil, errorsmod.Wrapf(authz.ErrNoAuthorizationFound, "failed to update grant with key %s", string(skey))
+				return nil, errorsmod.Wrapf(authz.ErrNoAuthorizationFound,
+					"failed to get grant with given granter: %s, grantee: %s & msgType: %s ", sdk.AccAddress(granter), grantee, sdk.MsgTypeURL(msg))
```

Comment:
```suggestion
					"failed to get grant with given granter: %s, grantee: %s & msgType: %s ", granter, grantee, sdk.MsgTypeURL(msg))
```
Granter is already of the `sdk.AccAddress` type.